nilearn/datasets/tests/test_atlas.py

```diff
@@ -126,13 +126,15 @@ def test_get_dataset_dir():
     # Non writeable dir is returned because dataset may be in there.
     assert_equal(data_dir, no_write)
     assert os.path.exists(data_dir)
+    os.chmod(no_write, 0o600) # Enable write permissions back
```

Comment:
PEP8 (two spaces before inline comment). Also mention why you reenabling write permissions (to be able to delete it I guess)